.env.example

```diff
@@ -7,6 +7,7 @@ LAALAA_API_HOST = "https://laa-legal-adviser-api-staging.apps.live-1.cloud-platf
 ENVIRONMENT = "dev"
 FEATURE_FLAG_NO_MAP = False
 
+DB_NAME=fala_db
```

Comment:
This is already [the default in our settings](https://github.com/ministryofjustice/fala/pull/263/files#diff-75397501301f0e287528185a96d3452d3200007bd6c068d77cdd489e1a18c987R42), so I don't know if it's valuable to declare it here too